lib/features/about/app.version.table.widget.dart

```diff
@@ -104,23 +104,6 @@ class _AppVersionTableState extends State<AppVersionTable> {
                 ),
               ],
             ),
-            DataRow(
```

Comment:
🤔 I think that perhaps something simple, give a try to the spread operator like this:
```dart
            ..._contributors
                .map((contributor) => DataRow(cells: <DataCell>[
                      DataCell(
                        ThemedTextLink(displayText: contributor.name, hyperlink: 'mailto:${contributor.email}'),
                      ),
                      DataCell(
                        ThemedTextLink(displayText: 'Linked-in', hyperlink: contributor.linkedIn),
                      )
                    ]))
                .toList(),
```